src/sass/_pattern_navigation.scss

```diff
@@ -175,6 +175,24 @@
   padding-left: 4rem !important;
 }
 
+.sidenav-top-container {
+  height: 60vh;
+  overflow: scroll;
+  -ms-overflow-style: none; /* Internet Explorer 10+ */
+  overflow-x: hidden;
+  scrollbar-width: none; /* Firefox */
+}
+
+.sidenav-top-container::-webkit-scrollbar {
+  display: none; /* Safari and Chrome */
+}
+
```

Comment:
```suggestion
```
We should never hide the scrollbar, as users with those browsers expect it to be present.